lib/action_subscriber/configuration.rb

```diff
@@ -8,13 +8,18 @@ class Configuration
                   :host,
                   :hosts,
                   :mode,
+                  :pass,
                   :pop_interval,
                   :port,
                   :prefetch,
                   :publisher_confirms,
+                  :user,
+                  :scheme,
+                  :ssl,
```

Comment:
Hmmmm, it seems very weird to have a config option for `ssl`, but not actually support it. Can we do something like `settings.slice(:vhost, :user, :pass, ...)` so we just ignore the `ssl` part of it?